tf2/CMakeLists.txt

```diff
@@ -55,13 +55,13 @@ if(BUILD_TESTING)
   # Should not lint external code
   set(
     _linter_excludes
-    include/tf2/LinearMath/Matrix3x3.h
-    include/tf2/LinearMath/MinMax.h
-    include/tf2/LinearMath/QuadWord.h
-    include/tf2/LinearMath/Quaternion.h
-    include/tf2/LinearMath/Scalar.h
-    include/tf2/LinearMath/Transform.h
-    include/tf2/LinearMath/Vector3.h
+    include/tf2/LinearMath/Matrix3x3.hpp
```

Comment:
you should keep here both the `h` and the `hpp`

Reply:
> you should keep here both the `h` and the `hpp`

Agreed.